### ISSUE TITLE
The clipboard is not automatically cleared when KeePassXC passwords are copied

### ISSUE THREAD

**Issue by TigBuh99** [hluk/CopyQ#2495]
The clipboard is not automatically cleared after a password from KeePassXC is copied.
I have set the passwords from KeePassXC in its settings to be cleared after 60 seconds.

1. Copy a password from KeePassXC
3. It's copied to the CopyQ clipboard
4. After 60 seconds have elapsed it still remains on the clipboard

Expected behavior: It should be removed from the clipboard after 60 seconds.

**Version, OS and Environment**
``
CopyQ v.7.0.0
System:
  Kernel: 6.1.0-12-amd64 [6.1.52-1] arch: x86_64 bits: 64 compiler: gcc v: 12.2.0
    parameters: BOOT_IMAGE=/boot/vmlinuz-6.1.0-12-amd64 root=UUID=<filter> ro quiet splash
  Desktop: Xfce v: 4.18.1 tk: Gtk v: 3.24.36 info: xfce4-panel wm: xfwm v: 4.18.0 vt: 7
    dm: LightDM v: 1.26.0 Distro: MX-23_x64 Libretto September 18  2023 base: Debian GNU/Linux 12
    (bookworm
``

**Comment by TigBuh99** [comment]
I installed [KeePassXC Protector](https://github.com/hluk/copyq-commands/blob/master/Automatic/keepassxc-protector.ini) and what happens is that the KeePassXC password is copied, it's not shown on the CopyQ main window and then it's removed after the period that I set in KeePassXC settings. I think that in Windows I could see the password in the CopyQ main window and then it disappeared after the set time.

**Comment by TigBuh99** [comment]
> Not sure if I understand. Is there still any issue with this? CopyQ by itself does not ignore password managers, it needs to be handled using the commands (like "KeePassXC Protector").
I looked at my Windows installation of CopyQ. I have a command titled _Ignore *'Password'* window_. It has a command _copyq ignore_ and in the Advanced tab, Match items the setting for Window is _KeePass_. the action is _Remove item enabled_. there is a description at the bottom saying that _if current window title matches KeePass._

Anyway, I verified that in my Windows installation the KeePass password is never stored, even temporarily. So the behavior is the same as in Linux. Therefore there is no issue with CopyQ.
